lib/internal/simulation_manager.dart

```diff
@@ -28,7 +28,6 @@ class SimulationManager extends SimulationManagerBaseWithErrorChecks
   }
 
   Future<void> _createClient() async {
-    print("entered create client");
   }
```

Comment:
Shouldn't this be formatted as a one-liner, with closing curly brace ending on the same line?